CHANGELOG.md

```diff
@@ -5,6 +5,116 @@ All notable changes to this project will be documented in this file.
+- `enclave.type` configuration entry now only supports `Debug` or `Release`. Trusted Execution Environment platform should be specified via new `enclave.platform` configuration entry (`SGX`, `SNP` or `Virtual`) (#4569).
+
+#### Release artefacts
+
+- `ccf_unsafe` is now a separate project and package, rather than the same project and package with a decorated version, to prevent accidental misuse.
+- Release assets now include variants per TEE platform: `ccf_sgx_<version>_amd64.deb`, `ccf_snp_<version>_amd64.deb` and `ccf_virtual_<version>_amd64.deb`.
+- Docker images now include variants per TEE platform, identified via image tag: `:<version>-sgx`, `:<version>-snp` and `:<version>-virtual`.
+- `sandbox.sh` now accepts a `--consensus-update-timeout-ms` to modify the `consensus.message_timeout` value in each node's configuration. This can be used to alter multi-node commit latency.
+
+---
+
+### Auditor
+
+- Node and service PEM certificates no longer contain a trailing null byte (#3885).
+
+---
+
+### Client API
+
+- Added a `GET /node/service/previous_identity` endpoint, which can be used during a recovery to look up the identity of the service before the catastrophic failure (#3880).
+- `GET /node/version` now contains an `unsafe` flag reflecting the status of the build.
+- Added new recovery_count field to `GET /node/network` endpoint to track the number of disaster recovery procedures undergone by the service (#3982).
+- Added new `service_data_json_file` configuration entry to `cchost` to point to free-form JSON file to set arbitrary data to service (#3997).
+- Added new `current_service_create_txid` field to `GET /node/network` endpoint to indicate `TxID` at which current service was created (#3996).
+- Experimental support for HTTP/2 (#4010).
+- Generated OpenAPI now describes whether each endpoint is forwarded (#3935).
+- When running with `curve-id` set to `secp256r1`, we now correctly support temporary ECDH keys on curve `secp256r1` for TLS 1.2 clients.
+- Application-defined endpoints are now accessible with both `/app` prefix and un-prefixed, e.g. `GET /app/log/private` and `GET /log/private` (#4147), both of which can be forwarded if necessary (#4534).
+- Session consistency is now provided even across elections. If session consistency would be broken, the inconsistent request will return an error and the TLS session will be terminated.
+
+---
+
+### Dependencies
+
+- Updated PSW in images to 2.17.100.
+- Upgraded Open Enclave to 0.18.4.
+
+---
+
+### Documentation
+
+- The "Node Output" page has been relabelled as "Troubleshooting" in the documentation and CLI commands for troubleshooting have been added to it.
+
+---
+
+### Fixes
+
+- Fix issue with large snapshots that may cause node crash on startup (join/recover) if configured stack size was too low (#4566).
```

Comment:
I think this should be moved under the `Operations` section, and the `Fixed` subheading can be removed.

Reply:
https://github.com/microsoft/CCF/pull/4632/commits/c983985e8236a5cf3b4e28425a7b09f4b9098817